.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        node-version: [20, 18, 16, 14, 12, 10, 8, 6]
+        node-version: [20, 18]
```

Comment:
This also means we'll need to update line 74 in package.json to reflect that we do not support anything under Node 18.